Exiled.API/Features/Room.cs

```diff
@@ -68,6 +68,19 @@ public float LightIntensity
             set => FlickerableLightController.Network_lightIntensityMultiplier = value;
         }
 
+        /// <summary>
+        /// Gets or sets the color of the room's lights.
+        /// </summary>
```

Comment:
The property name and description are a bit deceiving. This does not change the room's color, but rather the room's warhead light color, but that isn't mentioned anywhere. 

My main issue with this is that it interferes with warhead. I made a similar pr but closed it when I realized that it would cause unexpected behavior. The way it works needs to be fully documented.

Reply:
The thing is that the game itself calls this room color. The roomcolor command does the same thing, well internally its called Warhead color. I think its safe to call it roomcolor because that is what the base game command calls it.